### PR TITLE
Create RD subset pipeline

### PR DESCRIPTION
**Pull request by matentzn**
also closes https://github.com/monarch-initiative/mondo/issues/7704

### CODE REVIEW

File: src/ontology/mondo.Makefile [hunk 1]
```diff
@@ -266,19 +266,142 @@ $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIRep
 $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
 	wget "https://github.com/monarch-initiative/medgen/releases/latest/download/medgen-xrefs.robot.template.tsv" -O $@
 
+## FIX THIS BEFORE MERGING
```
Comment (twhetzel) [inline src/ontology/mondo.Makefile:269]: @matentzn what is supposed to be fixed here?

Comment (matentzn) [reply src/ontology/mondo.Makefile]: Oh yes. I doubled checked it, this has already been fixed on master branch, I deleted the goal (and comment)

**Comment by twhetzel** [comment]
This still fails despite being in the :
```
FAIL Rule ../sparql/qc/mondo/qc-animal-disease-rare.sparql: 1 violation(s)
entity,property,value
http://purl.obolibrary.org/obo/MONDO_0005676,http://purl.obolibrary.org/obo/mondo#rare,Animal disease in Rare subset
``

**Comment by matentzn** [comment]
@twhetzel checking out for today - I _think_ this PR is ready to go! We will discuss the more complicated part on the Mondo Ingest side more carefully.

File: src/ontology/mondo.Makefile [hunk 2]
```diff
@@ -266,79 +266,197 @@ $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIRep
...
+		query --update ../sparql/update/insert-gard-obsoletion-status.ru \
+		convert -f obo --check false -o $(SRC).obo
+	mv $(SRC).obo $(SRC) && make NORM && mv NORM $(SRC) && make deprecated_annotation_merging && make NORM && mv NORM $(SRC)
+
+##### NORD #########################
+
+.PHONY: update-nord
+update-nord:
+	make $(TMPDIR)/external/processed-nord.robot.owl -B
+	grep -vE '^(xref: NORD:|subset: nord_rare)' $(SRC) > $(TMPDIR)/mondo-edit.tmp || true
```
Comment (twhetzel) [inline src/ontology/mondo.Makefile:372]: Since NORD also includes xrefs, does this also need a query similar to `insert-gard-obsoletion-status.ru`?

**Review by twhetzel — APPROVED** [review]
@matentzn I have 1 question inline. If no further action is needed based on the question, please merge.

**Comment by matentzn** [comment]
> @matentzn I have 1 question inline. If no further action is needed based on the question, please merge.

I answered this on slack, adding here for the record only:

> Very important question. The more general version of this question is this:
Should we in all cases reject erroneous content from upstream sources, or sometimes attempt to repair it? In this case the question is: If the source provides an xref to an obsolete term, should we
Remove that xref during preprocessing and add it to the source report so they can fix it or
Fix that xref to mean: “this xref is to an obsolete term but no big deal” -
I personally thinks we should go for 1. I left the GARD as a single exception to that rule, as we dont have a very good update circle with them yet (at the moment they sporadically email updates to us), so I opted for repairing, but for all other sources I opted for (1). Let me know what you think!

File: src/ontology/mondo.Makefile (hunk 2 above)
Comment (matentzn) [reply src/ontology/mondo.Makefile:372]: See slack, for now, I think its good like this!

**Pull request merged**